### DISCUSSION TITLE
¡Contamos una historia juntas en el foro!

### DISCUSSION THREAD

**Discussion by KasturiSShah** [PatronesHermosos/design-thinking-1ra-generacion#2] [General]
Le invitamos a contar una historia una frase a la vez en nuestro foro con los demás alumnas. 

Le aconsejamos a usar este patron para contar la historia[ https://es.wikipedia.org/wiki/El_viaje_del_escritor#Sumario_del_contenido](https://es.wikipedia.org/wiki/El_viaje_del_escritor#Sumario_del_contenido) 

Redacte su contribución y haga clic en el botón verde que se llama “Comment” a continuación para publicar su contribución.

La primera frase es: 
**_El día estaba lleno de posibilidades._**

**Comment by jennyperez28** [comment]
Todas estas posibilidades formaban todo un árbol de caminos por tomar.

**Reply by Gaga1982** [reply]
La creatividad requiere tener el valor de desprenderse de las certezas.

**Reply by MaryAnnHernani** [reply]
El valor está dentro de cada uno de nosotros y es maravilloso cuando te das cuenta que puedes recorrer tus propios caminos.

**Comment by EdittTB** [comment]
Me senti con suerte y salí a caminar.

**Comment by Neri-hV** [comment]
y tomé el camino que creí que me abriría más posibilidades...

**Comment by Letty26** [comment]
Así que camine, abriendo en el camino un sinfín de posibilidades maravillosas.

**Comment by PilyRmz** [comment]
Esas posiblidades abrieron paso a una odisea, donde aprendí a no tener miedo de volver a empezar

**Comment by Letty26** [comment]
Así que inicie el camino, estudiando, participando y conociendo a personas maravillosas en él.
